tests/integration/crud/advanced/yaml/index-chunk.yml

```diff
@@ -0,0 +1,50 @@
+!CompoundIndexer
+components:
+  - !NumpyIndexer
+    with:
+      index_filename: vec.gz
+      metric: cosine
+    metas:
+      name: vecidx
+      workspace: $JINA_CRUD_ADVANCED_WORKSPACE
```

Comment:
as per separated workspace name, make sure that settimg the workspace at component level and at ComppundIndexer level is coherent with where you expect ir files to land

Reply:
yes, it still works, but like this is more clear what it does